docs/MODULES.md

```diff
@@ -295,101 +384,76 @@ If you use this rule, there is no point in having any other Yggdrasil client rul
+### Known install bugs
 
-You'll see the peering information that will give you the address (ipv4 or ipv6) and credentials to connect to 
-the node. You must either select to use just the ipv4 config, or you could use both. Now that we have this info, 
-connect to your mesh device, and edit the following config file:
+At times Grafana will not start up properly during install and the dashboards will not install.  To install them manually run the following commands from the `prototype-cjdns-pi/scripts/grafana` folder
 
 ```
-$ sudo nano /etc/cjdroute.conf
+BASE_DIR=`pwd`
+curl --user admin:admin -X POST -H 'Content-Type: application/json' --data-binary "@$BASE_DIR/datasource.json" http://localhost:3000/api/datasources
+curl --user admin:admin -X POST -H 'Content-Type: application/json' --data-binary "@$BASE_DIR/dashboard.json" http://localhost:3000/api/dashboards/db
 ```
 
-This file contains everything that is required for CJDNS to run, so be careful not to remove anything else, unless
-you know what you are doing. You need to head down to this line:
-
-```
-// Nodes to connect to (IPv4 only).
-"connectTo":
-{
-```
+### Allow Anonymous access
 
-This is where you input the ipv4 address. There is also a ipv6 field:
+To allow read-one guest access without a password edit `/etc/grafana/grafana.ini`
 
-```
-// Nodes to connect to (IPv6 only).
-"connectTo": 
-{
-```
+Uncomment/Change settings as follows
 
-Insert the respective code, and the save (ctrl+X to save, then ctrl+S to confirm file name, then ENTER to confirm changes).
-your code should look somewhat like this:
 ```
-                // Nodes to connect to (IPv4 only). 
-                "connectTo":
-                 {
-                               "159.203.5.91:30664": {
-                                 "peerName": "deprecated.systems",
-                                 "login": "tomesh-public",
-                                 "password": "iuw4nklm3j89qno876ef2jabpvlg1j0",
-                                 "publicKey": "2scyvybg4qqms1c5c9nyt50b1cdscxnr6ycpwsxf6pccbmwuynk0.k"
-                               }
-                 }
+[auth.anonymous]
+# enable anonymous access
+enabled = true
+# specify organization name that should be used for unauthenticated users
+org_name = Main Org.
+# specify role for unauthenticated users
+org_role = Viewer
 ```
 
-Next you should restart CJDNS with a `sudo systemctl restart cjdns` command. This will reload CJDNS
-with the new config file. Run a `status` command on your node, and make sure when it prints out
-the text, that CJDNS is green with the text `[ACTIVE]`. if so, you have successfully connected to the remote peer,
-if it says `[INACTIVE]`, then there might be a typo in your config file. Make sure its formatted correctly (the
-config file is written using JSON).
-
-### Yggdrasil
+## Prometheus
```

Comment:
A description and link explaining what it is and what it does on our nodes should be here.